gpt-survey-summarizer/templates/cronjob.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.databaseDumpExport.enabled }}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "gpt-survey-summarizer.fullname" . }}-database-dump-export
```

Comment:
**nitpick:** I suggest to call this job `database-dump-exporter`, in order to convey the idea of an "action" more than an "object". What do you think?

Reply:
I am reopening this conversation because I looks like it is not applied.

Yes, I was suggesting `name: {{ include "gpt-survey-summarizer.fullname" . }}-database-dump-exporter`.

This would mean the code should be change everywhere from `export` to `exporter`, like in the `values.yaml`, `databaseDumpExport` would change to `databaseDumpExporter`.

Feel free to decide to apply this suggestion or not, if you don't see much benefits versus the work.